Add Secoda to data catalog listing

semanticentity opened the pull request:
## Summary
This PR adds Secoda as a new listing to the awesome-data-catalogs repository.

## Changes Made
- ✅ Added Secoda to the high-level feature comparison table with full feature support across all categories
- ✅ Added Secoda to the table of contents in the 'Other Proprietary DCs' section
- ✅ Created a comprehensive detailed section for Secoda including:
  - Website link and description
  - Complete feature matrix
  - Detailed 'More features' section with 19+ capabilities
  - Extensive list of 70+ supported data sources

## Details
Secoda is a comprehensive data discovery and cataloging platform that provides:
- Automated data discovery and lineage tracking
- AI-powered features including agents and recommendations
- Advanced collaboration and governance capabilities
- Support for 70+ data sources including Snowflake, BigQuery, dbt, Tableau, and more

## Compliance
- ✅ Follows all contribution guidelines from CONTRIBUTING.MD
- ✅ Maintains consistent formatting with existing entries
- ✅ Added to bottom of relevant category as requested
- ✅ Single tool submission (1 new tool = 1 PR)
- ✅ Proper capitalization and formatting

All information has been thoroughly reviewed and is accurate.

semanticentity committed:
Add Secoda to data catalog listing

- Added Secoda to high-level feature comparison table with full feature support
- Added Secoda to table of contents in Other Proprietary DCs section
- Added detailed Secoda section with comprehensive feature list and data sources
- Follows contribution guidelines for new tool submissions

RamanDamayeu commented:
Hi @semanticentity thanks for contributing to the repo! 
Just recently we've refined a bit structure of the main table of tools/platforms, and also put more effort to compare AI and GenAI capabilities of the tools and platforms. I believe that approach also aligns with Secoda - as stated "The AI platform for data and analytics".
May I please ask you to update your description with a slightly updated scheme?
Thank you!

semanticentity committed:
feat(secoda): Enhance AI capabilities and positioning

- Update main description to emphasize AI/ML capabilities
- Add 6 new AI-specific features including multi-agent system
- Improve technical details of AI functionality
- Maintain all existing data sources and features

semanticentity committed:
feat(secoda): Add AI-powered data catalog with 142+ integrations

- Add Secoda as AI-powered data discovery platform
- Emphasize 24/7 AI data analyst positioning
- Include 9 specialized AI agents (Analysis, Automation, Search, Memory, etc.)
- Add natural language querying and AI-powered query generation
- Include intelligent error recovery and context-aware AI
- Updated to 142+ data source integrations based on sitemap analysis
- Full feature matrix with all capabilities marked

semanticentity committed:
Resolve merge conflicts and update Secoda section with AI features

semanticentity committed:
feat: Add Secoda to all GenAI Readiness Features matrices

- Added Secoda to AI & Semantic Metadata Enrichment matrix with full AI capabilities
- Added Secoda to Semantic Translation & Business Glossaries matrix
- Added Secoda to Technical Metadata & Query Analytics matrix
- Added Secoda to Data Quality, Observability & Governance matrix
- Added Secoda to High-Level Feature Comparison table
- All entries show full feature support reflecting Secoda's AI-powered platform capabilities

semanticentity committed:
feat: Add Secoda to GenAI readiness matrices with proper formatting

semanticentity committed:
Resolve merge conflicts in Table of Contents

semanticentity committed:
Fix remaining merge conflict markers in High-Level Feature Comparison table

semanticentity commented:
Hi @RamanDamayeu,

Thanks again for your review. I updated the Secoda section with the new AI and GenAI readiness features, resolved all merge conflicts, and aligned the formatting with the latest structure. The PR is now clean and hopefully ready to go.

Appreciate your time!

RamanDamayeu commented on README.md:
```diff
@@ -249,21 +253,23 @@ Essential features for maintaining data reliability, trustworthiness, observabil
 | Microsoft Purview                  | Microsoft                                           | Cloud         | -                                                           | [azure.microsoft.com](https://azure.microsoft.com/en-us/products/purview) | ✔️           | ✔️              | ✔️                 | ✔️          | ✔️                    | ✔️            | ✔️                       | ✔️                ...
 
 
+>>>>>>> secoda-genai-updates
```
Hi @semanticentity, could please ask to check formatting here (I believe it came from merging)
